client/control/components/ColumnMenu.js

```diff
@@ -41,16 +41,14 @@ class ColumnMenu extends React.Component {
    * simply updates the component's column state,
    * and notifies the parent of what's selected
    *
-   * Note: it seems a little fragile to use the index for this,
-   * may be useful to switch to something more identifying.
    *
-   * @param  {Number}  columnIndex Integer, index of relevant column
-   * @param  {Boolean} isActive    Is the column now active?
+   * @param  {String}  columnValue  Value of the column ('name', 'endTime', etc)
```

Comment:
Using "value" for this seems a bit strange. It makes sense in the context of check boxes, because HTML forms are odd, but this is a pretty different context. I think this will end up being confusing in the future. Can we use another name? For the human readable one, maybe something like "display", "label", or "title", and for the machine readable one "slug" or "id"?

(If you're not familiar with the term slug, it is sort of a Django-ism. Usually it is just a url-ified version of a title, like "Hello, World!" becoming "hello-world")